releases/mainnet/1.9.3/index.html

```diff
@@ -0,0 +1,25 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.3</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.3</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.3/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmeMhendy8pyeBopnVDexoJfWXvBhZ3rdQH3yb9UDaphaD">Contract addresses (IPFS hosted)</a></li>
```

Comment:
This lacks the pips so will need an update.

Reply:
done